src/Components/Modal/ModalCta.tsx

```diff
@@ -0,0 +1,57 @@
+import Colors from "Assets/Colors"
+import InvertedButton from "Components/Buttons/Inverted"
+import React, { SFC } from "react"
+import styled from "styled-components"
+import { media } from "../Helpers"
+
+export interface CtaProps {
+  isFixed: boolean
+  text: string
+  onClick: () => void
+}
+
+export const ModalCta: SFC<{
+  cta?: CtaProps
+  hasImage?: boolean
+  onClose: () => void
+}> = props => {
+  const {
+    cta: { isFixed, onClick, text },
+    hasImage,
+    onClose,
+  } = props
+
+  return (
+    <Cta isFixed={isFixed} hasImage={hasImage}>
+      <InvertedButton onClick={onClick || onClose}>{text}</InvertedButton>
+    </Cta>
+  )
+}
+
+const Cta = styled.div.attrs<{ isFixed?: boolean; hasImage?: boolean }>({})`
+  padding: 20px 0 30px 0;
+  button {
+    margin: 0;
+    width: 100%;
+  }
+  ${props =>
+    props.isFixed &&
+    `
+    position: absolute;
+    bottom: 0;
+    right: 40px;
+    left: ${props.hasImage ? "calc(50% + 40px)" : "40px"};
+    background: white;
+    border-top: 1px solid ${Colors.grayRegular};
+  `};
+
+  ${media.sm`
+    padding-bottom: 20px;
+    ${props =>
+      props.isFixed &&
+      `
+      right: 20px;
+      left: 20px
```

Comment:
Something you _could_ use in some of these places in lieu of hardcoding pixel values is to convert them to [spacing units](https://www.notion.so/artsy/Spacing-93eeaed9fdf9480099fec7094fd1b9f3#3e3c921525b7468f93cd2c2c1a662b6d). Like colors, Palette has first class support for spacing units. 

```typescript
import { space } from "@artsy/palette"

`
right: ${space(2)}px
left: ${space(2)}px
`
```

It might seem a little unnecessary from the outset, but over time it'll help us unify how we handle space between things. With that said, I don't think it needs to be tackled now or in this PR. It is an option though.